src/arrays/ArrayUtil.java

```diff
@@ -0,0 +1,140 @@
+package arrays;
+
+import java.util.Random;
+import java.util.Arrays;
+
+public class ArrayUtil {
+    /**
+     * Create an array with the given size, fill it with the different values and print it.
+     */
+    public static void createArray(int size) {
+        Random random = new Random();
+        int[] array = new int[size];
+        for (int i = 0; i < size; i++) {
+            array[i] = random.nextInt();
+            System.out.print(array[i] + " ");
+        }
+    }
+
+    /**
+     * Create two dimensional array with the given sizes, fill it with different values and print it.
+     */
+    public static void create2dArray(int x, int y) {
+        int[][] array1 = new int[x][y];
+        for (int i = 0; i < x; i++) {
+            for (int j = 0; j < y; j++) {
+                array1[i][j] = i + j;
+                System.out.print(array1[i][j] + " ");
+            }
+            System.out.println();
+
+        }
+    }
+
+    /**
+     * Add the given value at the beginning of the given array
+     *
+     * @return new array.
+     */
+    public static int[] addFirst(int[] array, int valueToAdd) {
+        int[] secondArray = new int[array.length + 1];
+        int index = 1;
+        secondArray[0] = valueToAdd;
+        for (int i = 0; i < array.length; i++) {
+            secondArray[index] = array[i];
+            index++;
+        }
+        return secondArray;
+    }
+
+    /**
+     * Implement a method that will check if the given value contains in the array
+     *
+     * @return true if the array contains the given value, otherwise return false.
+     */
+    public static boolean contains(int[] array, int value) {
+        for (int i = 0; i < array.length; i++) {
+            if (value == array[i]) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
+     * @param array An array to search in.
+     * @param value The value to search for.
+     * @return the first index of value in the array, if array does not contain given element return -1.
+     */
+    public static int indexOf(int[] array, int value) {
+        for (int i = 0; i < array.length; i++) {
+            if (value == array[i]) {
+                return i;
+            }
+        }
+        return -1;
+    }
+
+    /**
+     * Remove the element by the given index.
+     * Print new array
+     */
+    public static void remove(int[] array, int index) {
+        int[] newarray = new int[array.length - index];
```

Comment:
You misunderstood the meaning of the problem.
You need to remove the element at the given index.
You can write for example like this
   int secondArray = 0;
        if (array.length > 0 && array != null) {
            int[] newArray = new int[array.length - 1];
            for (int i = 0; i < array.length; i++) {
                if (i != index) {
                    newArray[secondArray++] = array[i];
                }
            }
            System.out.println(Arrays.toString(newArray));
        }